core/api/src/services/notifications/index.ts

```diff
@@ -699,6 +700,7 @@ export const NotificationsService = (): INotificationsService => {
         updateEmailAddress,
         removeEmailAddress,
         removePushDeviceToken,
+        triggerMarketingNotification,
```

Comment:
did you check how this methods params are serialized in honeycomb? we can have thousand of items in the params and can be problematic.. if it is please put this method outside the wrappers

Reply:
Will remove it from tracing.

Reply:
this wont work... we decompose object params